control-plane/stor-port/src/types/v0/store/nexus.rs

```diff
@@ -107,6 +108,8 @@ pub struct NexusSpec {
     #[serde(default)]
     /// Hosts allowed to access the nexus.
     pub allowed_hosts: Vec<HostNqn>,
+    /// rebuild policy for this nexus
+    pub rpolicy: RuleSet,
```

Comment:
Let's not add these in the objects until we're set on how the policy should look like, otherwise these may be breaking changes.
We can take a default policy for now.

Reply:
ok. Commented the field with a TODO tag. The get call for duration will return default 10mins for now.